scripts/maestro/run-ios-live-update.sh

```
@@ -0,0 +1,538 @@
+EXAMPLE_DIR="$ROOT_DIR/example-app"
+ARTIFACT_DIR="$ROOT_DIR/.maestro-artifacts"
+RESULTS_DIR="${CAPGO_MAESTRO_RESULTS_DIR:-$ROOT_DIR/maestro-results-ios-live-update}"
+HOST_SERVER_PORT="${CAPGO_MAESTRO_PORT:-3192}"
+HOST_SERVER_URL="${CAPGO_MAESTRO_HOST_BASE_URL:-http://127.0.0.1:${HOST_SERVER_PORT}}"
+APP_ID="app.capgo.updater"
+SCENARIO_SELECTION="${1:-all}"
+SERVER_PID=""
+SIMULATOR_BOOT_TIMEOUT_SECONDS="${CAPGO_MAESTRO_IOS_BOOT_TIMEOUT_SECONDS:-180}"
+MAESTRO_TIMEOUT_SECONDS="${CAPGO_MAESTRO_TIMEOUT_SECONDS:-360}"
+MAESTRO_CLI_NO_ANALYTICS="${MAESTRO_CLI_NO_ANALYTICS:-1}"
+MAESTRO_DRIVER_STARTUP_TIMEOUT_VALUE="${MAESTRO_DRIVER_STARTUP_TIMEOUT:-300000}"
```

Shellcheck (0.11.0)

[warning] MAESTRO_DRIVER_STARTUP_TIMEOUT_VALUE appears unused. Verify use (or export if used externally).

(SC2034)